packages/material-ui/src/TableBody/TableBody.test.js

```diff
@@ -1,33 +1,35 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import { getClasses, createMount, createClientRender, describeConformance } from 'test/utils';
+import { createMount, createClientRender, describeConformanceV5 } from 'test/utils';
 import TableBody from './TableBody';
 import Tablelvl2Context from '../Table/Tablelvl2Context';
+import classes from './tableBodyClasses';
 
 describe('<TableBody />', () => {
   const mount = createMount();
-  let classes;
   const render = createClientRender();
 
   function renderInTable(node) {
     return render(<table>{node}</table>);
   }
 
-  before(() => {
-    classes = getClasses(<TableBody />);
-  });
-
-  describeConformance(<TableBody />, () => ({
+  describeConformanceV5(<TableBody />, () => ({
     classes,
     inheritComponent: 'tbody',
     mount: (node) => {
       const wrapper = mount(<table>{node}</table>);
       return wrapper.find('table').childAt(0);
     },
-
+    wrapRender: (node) => {
```

Comment:
Agree with Marija, `render` sounds better
```suggestion
    render: (node) => {
```

Reply:
Am I still ok to use `wrapRender` in the conformance tests as the name would conflict with the normal `render`?